packages/camera/camera_android/lib/src/utils.dart

```diff
@@ -5,6 +5,8 @@
 import 'package:camera_platform_interface/camera_platform_interface.dart';
 import 'package:flutter/services.dart';
 
+import 'messages.g.dart';
+
 /// Parses a string into a corresponding CameraLensDirection.
 CameraLensDirection parseCameraLensDirection(String string) {
```

Comment:
Interesting, the IDE did not show this anywhere in its "Problems" tab. Removing.

Reply:
That's because from a language standpoint this is public code that anyone could call. By Dart convention, direct inclusion of `src/` files from other packages is not something you should do, and if you do it you are subjecting yourself to breakage without warning (because people assume clients aren't doing that for semver purposes).

So from our standpoint it's dead code, but from the analyzer's standpoint it's just a public method that we don't happen to also use internally.